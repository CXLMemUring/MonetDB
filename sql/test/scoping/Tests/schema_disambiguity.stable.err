stderr of test 'schema_disambiguity` in directory 'sql/test/scoping` itself:


# 11:04:05 >  
# 11:04:05 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-219686" "--port=33186"
# 11:04:05 >  

MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT tmp."outer"; -- unkown variable within sys
ERROR = !SELECT: no such column 'tmp.outer'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT "Avar";	-- unknown
ERROR = !SELECT: identifier 'Avar' unknown
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT "Bvar";	-- unknown
ERROR = !SELECT: identifier 'Bvar' unknown
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT A."outer"; -- unknown
ERROR = !SELECT: no such column 'a.outer'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT B."outer"; -- unknown
ERROR = !SELECT: no such column 'b.outer'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT B."Avar"; -- unknown
ERROR = !SELECT: no such column 'b.Avar'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT A."Bvar"; -- unknown
ERROR = !SELECT: no such column 'a.Bvar'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT "outer"; -- unkown
ERROR = !SELECT: identifier 'outer' unknown
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT "Bvar";	-- unknown
ERROR = !SELECT: identifier 'Bvar' unknown
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT A."outer"; -- unknown
ERROR = !SELECT: no such column 'a.outer'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT B."outer"; -- unknown
ERROR = !SELECT: no such column 'b.outer'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT B."Avar"; -- unknown
ERROR = !SELECT: no such column 'b.Avar'
CODE  = 42S22
MAPI  = (monetdb) /var/tmp/mtest-219686/.s.monetdb.33186
QUERY = SELECT A."Bvar"; -- unknown
ERROR = !SELECT: no such column 'a.Bvar'
CODE  = 42S22

# 11:04:05 >  
# 11:04:05 >  "Done."
# 11:04:05 >  

