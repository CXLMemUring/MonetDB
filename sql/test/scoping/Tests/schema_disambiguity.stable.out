stdout of test 'schema_disambiguity` in directory 'sql/test/scoping` itself:


# 11:04:05 >  
# 11:04:05 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-219686" "--port=33186"
# 11:04:05 >  

#CREATE SCHEMA A;
#CREATE SCHEMA B;
#DECLARE "outer" string;
#SET "outer" = 'outer';
#SELECT "outer";
% . # table_name
% single_value # name
% clob # type
% 5 # length
[ "outer"	]
#SELECT sys."outer"; -- same as above
% . # table_name
% single_value # name
% clob # type
% 5 # length
[ "outer"	]
#SET SCHEMA A;
#DECLARE "Avar" string;
#SET "Avar" = 'Avar';
#SELECT "Avar";
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Avar"	]
#SET SCHEMA B;
#DECLARE "Bvar" string;
#SET "Bvar" = 'Bvar';
#SELECT "Bvar";
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Bvar"	]
#SET SCHEMA sys;
#SELECT "outer"; -- should be known
% . # table_name
% single_value # name
% clob # type
% 5 # length
[ "outer"	]
#SELECT sys."outer";
% . # table_name
% single_value # name
% clob # type
% 5 # length
[ "outer"	]
#SELECT A."Avar"; -- known
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Avar"	]
#SELECT B."Bvar"; -- known
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Bvar"	]
#SET SCHEMA A;
#SELECT "Avar";	-- known
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Avar"	]
#SELECT sys."outer";
% . # table_name
% single_value # name
% clob # type
% 5 # length
[ "outer"	]
#SELECT A."Avar"; -- known
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Avar"	]
#SELECT B."Bvar"; -- known
% . # table_name
% single_value # name
% clob # type
% 4 # length
[ "Bvar"	]
#SET SCHEMA "sys";
#DROP SCHEMA A CASCADE;
#DROP SCHEMA B CASCADE;

# 11:04:05 >  
# 11:04:05 >  "Done."
# 11:04:05 >  

